src/PaprController.sol

```diff
@@ -349,16 +349,19 @@ contract PaprController is
     /// @inheritdoc IPaprController
     function setPool(address _pool) external override onlyOwner {
         _setPool(_pool);
+        emit PoolChange(_pool);
```

Comment:
is `Change` idiomatic? Feel like I don't see that used much. Maybe `UpdatePool`?